specs/core/core-vocab.ttl

```diff
@@ -24,11 +24,14 @@ oslc:
   a owl:Ontology ;
   dcterms:title "The OSLC Core Vocabulary" ;
   dcterms:description "All vocabulary URIs defined in the OSLC Core namespace."^^rdf:XMLLiteral ;
-  dcterms:modified "2020-06-11"^^<http://www.w3.org/2001/XMLSchema#date> ;
+  rdfs:label "Core" ;
+  vann:preferredNamespacePrefix "oslc" ;
   dcterms:publisher <https://open-services.net/about/> ;
+  dcterms:issued "2020-08-06"^^<http://www.w3.org/2001/XMLSchema#date> ;
+  dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
   dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/core/core-vocab.ttl> ;
-  vann:preferredNamespacePrefix "oslc" ;
-  rdfs:label "Core" .
+  dcterms:isPartOf <https://oslc-op.github.io/oslc-specs/specs/core/oslc-core.html> ;
+  dcterms:hasVersion "PS01" .
```

Comment:
let's remove this property until the spec is approved for publishing.

Reply:
Fixed